django/db/migrations/executor.py

```diff
@@ -162,6 +177,17 @@ def _migrate_all_backwards(self, plan, full_plan, fake):
         for migration, _ in plan:
             self.unapply_migration(states[migration], migration, fake=fake)
 
+        # Generate a new state representing the remaining applied migrations.
+        state = self._create_project_state()
```

Comment:
Can't you reuse something from `states`? The respective state should already be in there. Then call `del state.__dict__['apps']` and re-render.